### DISCUSSION TITLE
Cropper replace throwin an exception, after second call

### DISCUSSION THREAD

**Discussion by justasxz** [CropperBlazor/Cropper.Blazor#378] [Q&A] [answered]
**Describe the bug**
The bug is as follows. I have a simple blazor hybrid application, that changes brightness with slider, meaning that Source is updated on every slider movement, I use MagickImage for that. I tried to combine it with blazor.cropper, I switched from <img> to <CropperComponent>, and it stopped updating when the source changed. After some reading I found that I would need to call 
await cropper.ReplaceAsync(SelectedImage, true);
And it works fine the first time, it updated the brightness and keeps the cropper, even if a little bit slow. But after the second brightness change it throws the following exception:

Cannot set properties of undefined (setting 'src')
TypeError: Cannot set properties of undefined (setting 'src')
    at n.replace (https://0.0.0.0/_content/Cropper.Blazor/cropper.min.js:10:22903)
    at CropperDecorator.replace (https://0.0.0.0/_content/Cropper.Blazor/cropperJsInterop.min.js:1:1746)
    at https://0.0.0.0/_framework/blazor.webview.js:1:3050
    at new Promise (<anonymous>)
    at g.beginInvokeJSFromDotNet (https://0.0.0.0/_framework/blazor.webview.js:1:3007)
    at https://0.0.0.0/_framework/blazor.webview.js:1:48020
    at EventTarget.<anonymous> (<anonymous>:7:62)
    at EmbeddedBrowserWebView.<anonymous> (<anonymous>:1:43721)

I believe this is a bug, but maybe I am missing something ? It feels like REplaceSync doesn't fully replace the component

**Comment by MaxymGorn** [comment, marked as answer]
Hi @justasxz. Could you glance to the official [docs](https://cropperblazor.github.io/examples/replacing#simple-usage).
![Screenshot_20240803_182635_Chrome](https://github.com/user-attachments/assets/370b0e20-a42a-4379-9c32-0ba6022065a0)
If it isn't help you. Could you provide example of project with that problem?
